sidebars.js

```diff
@@ -111,6 +111,11 @@ module.exports = {
                 },
               ],
             },
+            {
+              "Additional features": [
```

Comment:
We should not hide it under "Additional features". This could mean everything that we haven't found the right place for. It should be more like "Advanced modeling"

Reply:
What about the "Collaboration", "Milestones", "Token Simulation" pages? I think the section is getting too big.

Reply:
Good catch @CatalinaMoisuc, we could put "Token Simulation" under "Validate your models", and "Milestones" could belong to somewhere else, or both under "Advanded modeling". We shouldn't do it in the scope of this issue, but a general overhaul could help us to keeping a nice-to-read documentation. I would recommend that there should not be more than 7 parent sections

Reply:
@christian-konrad What about the "Collaboration", "Milestones", "Token Simulation" pages? I think the section is getting too big.

Reply:
Makes sense. Once we have these changes out, we can open a new PR to reorganize the pages. cc @nithinssabu

Reply:
https://github.com/camunda/web-modeler/issues/1777